app/views/playlists/index.html.erb

```diff
@@ -53,12 +53,9 @@ Unless required by applicable law or agreed to in writing, software distributed
         <div> </div>
       <% end %>
       <div>
-        <%= link_to(new_playlist_path) do %>
-          <span class="btn btn-primary outline_on">
-            Create New Playlist
-          </span>
-        <% end %>
-	      <% if Settings['variations'].present? %>
+	<%# pass body to link_to to avoid block form of link_to which creates an odd underline that overflows the button %>
+        <%= link_to('<span class="btn btn-primary outline_on">Create New Playlist</span>', new_playlist_path) %>
```

Comment:
This is html_encoding the link text. Maybe needs `.html_safe` added to the span string.